paddle/fluid/framework/ir/mkldnn/mkldnn_pass_util.h

```diff
@@ -155,6 +155,26 @@ static void GetInfoFromTheFirstOp(ir::Graph* graph,
   }
 }
 
+static void ConvertToFusedOp(OpDesc* op) {
```

Comment:
Why did you declare this function as static?
```suggestion
inline void ConvertToFusedOp(OpDesc* op) {
```
It should be inline IMHO.
https://stackoverflow.com/questions/8406800/when-should-i-write-the-keyword-static-before-a-non-member-function/8406986#8406986